diff --git a/01_install_requirements.sh b/01_install_requirements.sh
index 4303ba2..53e3335 100755
--- a/01_install_requirements.sh
+++ b/01_install_requirements.sh
@@ -99,7 +99,7 @@ esac
 # during dnf upgrade
 sudo dnf -y install jq
 sudo python -m pip install yq
-yq -iy '.[3].dnf.nobest = "true"' ${METAL3_DEV_ENV_PATH}/vm-setup/roles/packages_installation/tasks/centos_required_packages.yml
+yq -i '.[3].dnf.nobest = "true"' ${METAL3_DEV_ENV_PATH}/vm-setup/roles/packages_installation/tasks/centos_required_packages.yml
 
 GOARCH=$(uname -m)
 if [[ $GOARCH == "aarch64" ]]; then
